Automattic/simplenote-macos
# Shortcut: Create note with content

### Fix
The final shortcut to add is the create new note with content intent.  This allows a user to specify some text into the shortcut and to create a new note in simplenote with that text. 

This text is pushed to a new note via the simperium api, so it will sync to all devices without having to open simplenote locally

relies on #1170 
Fixes #1145 

### Test
1.  Build and Run Simplenote, just to make sure that you have saved the extension token so the intents can upload to simperium
2. run the intents extension on the shortcuts app
3. add a new shortcut, find Simplenote, select Create note with content intent
4. Add some content to the intent and run the shortcut
5. open simplenote and confirm your new note was added.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> `RELEASE-NOTES.txt` was updated in 739f83f with:
> - Added shortcuts to Simplenote Mac!!
If the changes should not be included in release notes, add a statement to this section. For example:

> These changes do not require release notes.
